opencti-platform/opencti-front/src/utils/widget/widgetUtils.tsx

```diff
@@ -269,3 +271,20 @@ export const renderWidgetIcon = (key: string, fontSize: 'large' | 'small' | 'med
       return <div />;
   }
 };
+
+export const isDataSelectionNumber = (type: string, dataSelection: WidgetDataSelection[]) => {
```

Comment:
```suggestion
export const isDataSelectionNumberValid = (type: string, dataSelection: WidgetDataSelection[]) => {
```
I think it's better that way to explain what the function does, it does not check if there is a selection number but if it is valid